src/spec-converter.js

```diff
@@ -68,11 +67,11 @@ export class SpecConverter {
 
             // leaf elements should inherit coordinates properties
             if (root && !childUnit.hasOwnProperty('units')) {
-                childUnit = _.defaults(childUnit, {x: root.x, y: root.y});
+                childUnit = utils.defaults(childUnit, {x: root.x, y: root.y});
 
-                var parentGuide = utils.clone(root.guide || {});
-                childUnit.guide.x = _.defaults(childUnit.guide.x || {}, parentGuide.x);
-                childUnit.guide.y = _.defaults(childUnit.guide.y || {}, parentGuide.y);
+                var parentGuide = root.guide || {};
```

Comment:
I'll add it back but the deep clone is something that should be reviewed and use scarcely since it currently has a huge performance impact in the library